frontend/src/scenes/error-tracking/queries.ts

```diff
@@ -155,11 +155,9 @@ export const errorTrackingGroupEventsQuery = ({
     const group = filterGroup.values[0] as UniversalFiltersGroup
     const properties = group.values as AnyPropertyFilter[]
 
-    const where = [
-        `has(${stringifyFingerprints(
-            fingerprints
-        )}, JSONExtract(ifNull(properties.$exception_fingerprint,'[]'),'Array(String)'))`,
-    ]
+    // TODO: fix this where clause. It does not take into account the events
+    // associated with issues that have been merged into this primary issue
+    const where = [`eq(${issueId}, properties.$exception_issue_id`]
```

Comment:
Need to come back to this but think it's ok for now. Basically we piggy back on top of the `EventsQuery` to fetch individual exceptions for an issue. Because we no longer have all the `merged_fingerprints` associated with an issue model in PG we cannot filter the query.

I _think_ what we'll end up doing here is returning `issue_ids` as a field on the `ErrorTrackingIssueQueryResponse` object (computed as `groupUniqArray(properties.$exception_issue_id)`) which can be reused to fetch individual events. I just want to confirm that the `groupUniqArray` doesn't make the query slow

Reply:
We'll need to hit the overrides table here, right? Same kind of query as getting the events for a person profile?